roles/kubernetes-apps/policy_controller/calico/templates/calico-kube-cr.yml.j2

```diff
@@ -72,8 +81,6 @@ rules:
       - get
       - create
       - update
-{% endif %}
```

Comment:
Are you sure about that ? the part below isn't needed when not in kdd mode ?

Reply:
Ok will add it back

Reply:
> Are you sure about that ? the part below isn't needed when not in kdd mode ?

I was a bit quick. Access to kubecontrollersconfigurations is only needed in kdd mode. See manifests at:
https://docs.projectcalico.org/getting-started/kubernetes/self-managed-onprem/onpremises
I searched for kubecontrollersconfigurations in the etcd manifest.
Do you still want the version check?

Reply:
All good that way I'd say